examples/features/externally-connected-points/externallyconnectedpoints01.fixture.tsx

```diff
@@ -0,0 +1,175 @@
+        "source_net_0",
+        "pcb_smtpad_0",
+        "pcb_port_0",
+        "pcb_smtpad_1",
+        "pcb_port_1",
+        "pcb_smtpad_2",
+        "pcb_port_2",
+        "pcb_smtpad_3",
+        "pcb_port_3",
+      ],
+    },
+    {
+      type: "rect",
+      layers: ["top"],
+      center: {
+        x: 3.5,
+        y: 0,
+      },
+      width: 0.6,
+      height: 0.6,
+      connectedTo: [
+        "pcb_smtpad_1",
+        "connectivity_net7",
+        "source_trace_3",
+        "source_port_3",
+        "source_trace_2",
+        "source_port_2",
+        "source_trace_1",
+        "source_port_1",
+        "source_trace_0",
+        "source_port_0",
+        "source_net_0",
+        "pcb_smtpad_0",
+        "pcb_port_0",
+        "pcb_smtpad_1",
+        "pcb_port_1",
+        "pcb_smtpad_2",
+        "pcb_port_2",
+        "pcb_smtpad_3",
+        "pcb_port_3",
+      ],
+    },
+    {
+      type: "rect",
+      layers: ["top"],
+      center: {
+        x: -3.5,
+        y: 0,
+      },
+      width: 0.6,
+      height: 0.6,
+      connectedTo: [
+        "pcb_smtpad_2",
+        "connectivity_net7",
+        "source_trace_3",
+        "source_port_3",
+        "source_trace_2",
+        "source_port_2",
+        "source_trace_1",
+        "source_port_1",
+        "source_trace_0",
+        "source_port_0",
+        "source_net_0",
+        "pcb_smtpad_0",
+        "pcb_port_0",
+        "pcb_smtpad_1",
+        "pcb_port_1",
+        "pcb_smtpad_2",
+        "pcb_port_2",
+        "pcb_smtpad_3",
+        "pcb_port_3",
+      ],
+    },
+    {
+      type: "rect",
+      layers: ["top"],
+      center: {
+        x: -2.5,
+        y: 0,
+      },
+      width: 0.6,
+      height: 0.6,
+      connectedTo: [
+        "pcb_smtpad_3",
+        "connectivity_net7",
+        "source_trace_3",
+        "source_port_3",
+        "source_trace_2",
+        "source_port_2",
+        "source_trace_1",
+        "source_port_1",
+        "source_trace_0",
+        "source_port_0",
+        "source_net_0",
+        "pcb_smtpad_0",
+        "pcb_port_0",
+        "pcb_smtpad_1",
+        "pcb_port_1",
+        "pcb_smtpad_2",
+        "pcb_port_2",
+        "pcb_smtpad_3",
+        "pcb_port_3",
+      ],
+    },
+  ],
+  connections: [
+    {
+      name: "source_net_0",
+      pointsToConnect: [
+        {
+          pointId: "TEST_POINT_0",
+          x: 2.5,
+          y: 0,
+          layer: "top",
+          pcb_port_id: "pcb_port_0",
+        },
+        {
+          x: 3.5,
+          y: 0,
+          layer: "top",
+          pcb_port_id: "pcb_port_1",
+        },
+        {
+          x: -3.5,
+          y: 0,
+          layer: "top",
+          pcb_port_id: "pcb_port_2",
+        },
+        {
+          pointId: "TEST_POINT_1",
+          x: -2.5,
+          y: 0,
+          layer: "top",
+          pcb_port_id: "pcb_port_3",
+        },
+      ],
+      externallyConnectedPointIds: [["TEST_POINT_0", "TEST_POINT_1"]],
```

Comment:
CC @imrishabh18 this new property allows you to specify that two points are connected somewhere else, and therefore don't need to be "reconnected"